Gemfile

```diff
@@ -68,8 +68,10 @@ group :test do
   gem 'rspec-html-matchers'
   gem 'rubocop', require: false
   gem 'selenium-webdriver'
+  gem 'chromedriver-helper'
```

Comment:
Can you please remove this gem again from the Gemfile? It does not match the Selenium / Chrome setup we use for the feature specs on this project and having a quick look at the CI output for this PR, it looks also like this is the cause for it being 🔴 for feature specs.

I know that this gem is an alternative way to install Chromedriver, but I have only had bad experience with it and in general don't like the idea of _"misusing"_ gems to install system dependencies 😉 - it usually turns out to be a shortcut that's not worth it after all.
If you ran into any problems with setting up the dependencies for the app, just check the `README` and especially our [`TROUBLESHOOTING` guide](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/TROUBLESHOOTING.md#chromedriver), we have a dedicated section for Chromedriver there 📚